### PR TITLE
Implementation FieldMapper and ModelMapper #4029

### PR DESCRIPTION
**Pull request by ywj1352**
Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/mapper/FieldMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+    @Test
+    public void testSelectByPrimaryKey() {
+        FieldDO record = buildFieldDO();
+        int count = fieldMapper.insert(record);
+        assertThat(count, greaterThan(0));
+
+        FieldDO fieldDO = fieldMapper.selectByPrimaryKey(record.getId());
+        assertThat(fieldDO != null, equalTo(true));
```
Comment (moremind) [inline shenyu-admin/src/test/java/org/apache/shenyu/admin/mapper/FieldMapperTest.java:82]: ...assertTrue

Comment (ywj1352) [reply shenyu-admin/src/test/java/org/apache/shenyu/admin/mapper/FieldMapperTest.java:82]: change to assertNotNull

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/FieldMapper.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+@Mapper
+public interface FieldMapper {
+
+    /**
+     * insert record to table.
+     *
+     * @param record the record
+     * @return insert count
+     */
+    int insert(FieldDO record);
```
Comment (li-keguo) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/FieldMapper.java:35]: Avoid using the keyword record,The new jdk will have problems

**Review by li-keguo — APPROVED** [review]

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/FieldMapper.java (hunk 2 above)
Comment (li-keguo) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/FieldMapper.java:35]: see :https://docs.oracle.com/en/java/javase/15/language/records.html

**Review by li-keguo — APPROVED** [review]

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/FieldMapper.java (hunk 2 above)
Comment (dragon-zhang) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/FieldMapper.java:35]: > Avoid using the keyword record,The new jdk will have problems

agree

Comment (ywj1352) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/FieldMapper.java]: done

**Review by dragon-zhang — APPROVED** [review]

**Pull request merged**